README.md

```diff
@@ -23,18 +29,20 @@ Extra features ( require some additional setup, look into "extra_features" folde
 * Monitoring of blocked processes (needs a cron script on the host DB)
 * Monitoring of pg_stat_statements (needs pg_stat_statements extension to be enabled)
 * Cron Aggregations for speeding up sproc load and database size (useful when monitoring tens of instances)
+* Exporting of metrics data to InfluxDB where it can be used for custom charting/dashboarding for example with Grafana
 
-Note: gathering of single metrics is configurable per instance ("hosts" table), so if you don't need so much details you don't have to gather them
+Gathering of single metrics is configurable per instance (see "hosts" table), so if you don't need so much details you don't have to gather them.
 
-The performance data is gathered by a Java application querying the different PostgreSQL performance views. Gather intervals for the different metrics can be configured on a per host per metric basis, enabling more detailed monitoring for critical systems and less details for less important systems, thus reducing the amount of data gathered. Additionally you can configure sets of hosts to monitor from different Java processes, e.g., if you deploy to multiple locations with limited accessability.
+Metrics are gathered by a Java application, querying the different PostgreSQL performance views (pg_stat_*).
+Gather intervals for the different metrics can be configured on a per host per metric basis, enabling more detailed monitoring for critical systems and less details for less important systems, thus reducing the amount of data gathered. Additionally you can configure sets of hosts to monitor from different Java processes, e.g., if you deploy to multiple locations with limited accessability.
 
-The web frontend is a Python CherryPy standalone application. See the screenshot folder for basic examples. Chart visualization is rendered using JS flot library.
+The Web Frontend is a standalone Python + CherryPy application. See the "screenshots" folder for basic examples. Chart visualization is rendered using JS flot library.
```

Comment:
links would be welcome (both to the dir and http://www.flotcharts.org/  and http://www.cherrypy.org/